src/components/Context/context.spec.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { render } from 'ink-testing-library';
+import { ListContexts, ShowCurrentContext, AddContext, SetCurrent } from './Context';
+import { TestingVariables } from '../../constants';
+
+let testing = new TestingVariables();
+
+describe('listing contexts', () => {
+	test("should render error when no context file found", () => {
+		testing.deleteDummyContextFile();
+		let { lastFrame } = render(<ListContexts />);
+		expect(lastFrame()).toMatch("No contexts saved yet.");
```

Comment:
please create another followup issue to turn these messages into constants so we reuse them in components and tests, something like https://github.com/asyncapi/asyncapi-react/blob/master/library/src/constants.ts

no need to fix it now, not related to this pr